payments/management/commands/delete_discount_code.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management import BaseCommand
+
+from payments.models import Discount
+
+
+class Command(BaseCommand):
+    """
+    Deletes multiple discounts using the Discount IDs.
+    An example usage of this command: python manage.py delete_discount_code 1 2 3
+    """
+
+    help = "Deletes multiple discounts using the Discount IDs."
+
+    def add_arguments(self, parser) -> None:
+        """
+        Add arguments to the command parser.
+        """
+        parser.add_argument(
+            "discount_ids",
+            type=int,
+            nargs="+",
+            help="The IDs of the discounts to delete.",
+        )
+
+    def handle(self, **options) -> None:
+        """
+        Handle the deletion of discounts based on provided discount IDs.
+        """
+        discount_ids = options["discount_ids"]
+        discounts = Discount.objects.filter(id__in=discount_ids)
```

Comment:
Tried to delete a code that had been redeemed and got an error. This should deactivate the code if it's been redeemed, not delete it (and create a version).